sites/public/pages/listings.tsx

```diff
@@ -171,18 +175,33 @@ const ListingsPage = () => {
           </div>
         </Form>
       </Modal>
-      <div className="max-w-3xl mt-6 m-auto">
-        <LinkButton size={AppearanceSizeType.small} href="/eligibility/welcome">
+      <div className="max-w-3xl mx-8">
```

Comment:
Something is setting the buttons to the far left vs centered. Is this intended?
![image](https://user-images.githubusercontent.com/82653098/130674239-9bd60a7b-76ad-44c7-95fb-3ef4ad6523d6.png)

Reply:
Looks like these are now aligned to the left in the mocks.  Closing.